lib/components/banner/banner.css

```diff
@@ -0,0 +1,35 @@
+@charset "utf-8";
+
+#banner {
+  flex: 0 0 auto;
+  width: 100%;
+  height: var(--banner-height);
+  display: flex;
+  align-items: stretch;
+  flex-direction: row;
+  justify-content: center;
+
+  text-align: center;
+  color: #e0e0e0;
```

Comment:
maybe can we just make this white and also turn it into a variable?

Reply:
Ah missed this, fixed!